### PR TITLE
Add biggie to SSB

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: mods/seasonal/random-teams.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: "Buzzing of the Storm",
 				evs: {def:4, atk:252, spe:252}, nature: 'Jolly',
 			},
+			'Biggie': {
+				species: 'Snorlax', ability: 'Fur Coat', item: 'Leftovers', gender: 'M', //ask gender
+				moves: ['drainpunch', ['diamondstorm', 'kingsshield'][this.random(2)], ['knockoff', 'precipiceblades'][this.random(2)]], // ask about moveset
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: confirm moveset before this gets merged, generally hes just going to have to pair up moves into sets of possible choices like now or limit it to 3.

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -170,6 +170,23 @@ exports.BattleMovedex = {
...
+	// biggie
+	foodrush: {
+		accuracy: 100,
+		basePower: 100,
+		category: "Physical",
+		shortDesc: "Forces the target to switch to a random ally.",
+		id: "foodrush",
+		isNonstandard: true,
+		name: "Food Rush",
+		pp: 10,
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:182]: `noPPBoosts: true,`

Comment (CheeseMuffin) [reply mods/seasonal/moves.js:182]: Done but didnt show up for some reason

**Comment by HoeenCoder** [comment]
Um @CheeseMuffin where did you repo go?

I cant merge this/resolve conflicts.

**Pull request closed without merging**